tornado_rest_client/api.py

```diff
@@ -434,17 +434,15 @@ class RestClient(object):
     # timeout.
     TIMEOUT = None
 
-    # If the APi expects that you send a JSON body with data rather than
-    # passing url arguments, set this to true.
-    JSON_BODY = False
```

Comment:
Lets leave this in here for backwards compatibility - otherwise this breaks anyone using this code. I'm fine with you adding the parameter to the class instantiation on line 437 though, and you can put some logic in there to handle letting that take priority over JSON_BODY if its supplied.